sdcm/cluster.py

```diff
@@ -1439,7 +1439,7 @@ def install_scylla(self, scylla_repo):
                 """)
                 self.remoter.run('sudo bash -cxe "%s"' % install_debian_9_prereqs)
 
-            self.remoter.run('sudo apt-get upgrade -y')
+            self.remoter.run('sudo DEBIAN_FRONTEND=noninteractive apt-get --force-yes -o Dpkg::Options::="--force-confold" -o Dpkg::Options::="--force-confdef" upgrade -y')
```

Comment:
@amoskong Why do we perform an upgrade of all packages?

Reply:
It's not necessary, but it will make the whole upgrade (scylla upgrade) smoothly. We upgrade the system to avoid scylla upgrade is broken by problem of external packages.

When we upgrade Fedora OS from 29 to 30, they also recommend user to upgrade to latest Fedora 29 first.